File: Vjezbe/Vjezbe3/Zad1.py
import particle as prt

p1 = prt.Particle()
p1.set_in_cond(10, 60, 0, 0, 0.01)
p1.range()
print("Domet je {:.2f} metara".format(p1.x[-1]))
p1.plot_trajectory()
p1.range_analitical()
print("Analitički domet je {:.2f} metara".format(p1.d))

File: Vjezbe/Vjezbe7/Zad1.cpp
#include <iostream>
#include <Particle.h>

using namespace std;
float d1, t1, d2, t2;

int main(){
    Particle p1(10, 45, 0, 0);
    Particle p2(40, 60, 0, 0);
    d1 = p1.range();
    t1 = p1.time();
    d2 = p2.range();
    t2 = p2.time();
    cout << "Domet prvog tijela je " << d1 << " metara." << std::endl;
    cout << "Vrijeme gibanja prvog tijela je  " << t1 << " sekundi." << std::endl;
    cout << "Domet drugog tijela je  " << d2 << " metara." << std::endl;
    cout << "Vrijeme gibanja drugog tijela je  " << t2 << " sekundi." << std::endl;
}